src/Task.gren

```diff
@@ -390,12 +391,25 @@ execute task =
     command (Execute (map (\_ -> {}) task))
 
 
+{-| This is like `execute` where we want to give a command without being told how it went,
+but follow up with a command to run once the task completes.
```

Comment:
Not sure about this description. In a way we know what the task is going to do, as the task cannot fail and will end with a `Cmd` value.

`execute` is essentially "just run this task".
`executeCmd` is "run this task and send/trigger the resulting cmd".